src/lib/removeObjectNulls.ts

```diff
@@ -1,5 +1,29 @@
-import { curry, pickBy } from 'lodash/fp';
+const curry = (fn) => {
+  return function curried(this: unknown, ...args) {
+    if (args.length >= fn.length) {
+      return fn.apply(this, args);
+    } else {
+      return (...nextArgs) => {
+        return curried.apply(this, args.concat(nextArgs));
+      };
+    }
+  };
+};
 
-export default curry((collection: object) =>
-  pickBy((val) => val !== null, collection),
-);
+const pickBy = (predicate, collection) => {
+  return Object.keys(collection).reduce((result, key) => {
+    if (predicate(collection[key])) {
+      result[key] = collection[key];
+    }
+    return result;
+  }, {});
+};
+
+export default curry((collection) => pickBy((val) => val !== null, collection));
```

Comment:
Why are we using currying? Wouldn't `export default (collection) => pickBy((val) => val !== null, collection)` suffice?

Reply:
I'm recreating how the old MPDX did it, to ensure we get the same results. I've removed the curry function.